stellarphot/gui_tools/tests/test_seeing_profile.py

```diff
@@ -112,6 +124,97 @@ def test_seeing_profile_properties(tmp_path):
         assert profile_widget.aperture_settings.value == phot_aps
 
 
+def test_seeing_profile_save_apertures(tmp_path):
+    # Make sure that saving partial photometery settings works
+    os.chdir(tmp_path)
+
+    phot_settings = PhotometryWorkingDirSettings()
+
+    # There should be no saved settings...
+    with pytest.raises(ValueError, match="does not exist"):
+        phot_settings.load()
+
+    profile_widget = spf.SeeingProfileWidget(
+        camera=Camera(**TEST_CAMERA_VALUES), _testing_path=tmp_path
+    )
+
+    profile_widget.save()
+    assert not profile_widget.aperture_settings.savebuttonbar.unsaved_changes
+
+    settings = phot_settings.load()
+    assert settings.camera == Camera(**TEST_CAMERA_VALUES)
+    assert settings.photometry_apertures == PhotometryApertures(
```

Comment:
Is this the default settings created with `TEST_CAMERA_VALUES`?

Reply:
Yes -- it should have been saved when the SeeingProfileWidget was called, above.